client/packages/system/src/InventoryAdjustmentReason/Components/InventoryAdjustmentReasonSearchInput.tsx

```diff
@@ -48,7 +48,6 @@ export const InventoryAdjustmentReasonSearchInput: FC<
     <Box display="flex" flexDirection="row" width={120}>
       <Autocomplete
         autoFocus={autoFocus}
-        disabled={isDisabled}
```

Comment:
Type is never not defined, so no need to disable reason selector.

Reply:
Hmm what if there are no reasons ?

Reply:
I thought we already had a check for this upstream, but I guess not. I'll hide the row completely if Reasons are not defined.

Reply:
Okay, have made it disabled when no reasons, as per discussion.